contracts/bridges/BaseBridgeReceiver.sol

```diff
@@ -79,4 +79,14 @@ contract BaseBridgeReceiver {
 
         emit ProcessMessage(messageSender, targets, values, signatures, calldatas, eta);
     }
+
+    function executeTransaction(
+        address target,
```

Comment:
Wondering if we just make it easier to call `execute`. Instead of passing all these args again, maybe we just provide the hash of the action or some unique id. This would require the `BaseBridgeReceiver` to store a mapping of these actions, but makes it way easier for anyone to call `execute`.

Reply:
I'm open to it. the nice thing about reusing the same arguments are that

- you don't have to store anything on BaseBridgeReceiver
- you can pull all the required arguments directly off of the Timelock's `QueueTransaction` event

Reply:
That's true, it should be pullable from the `QueueTransaction` event.